packages/wrangler/bin/wrangler.js

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
-const { spawn } = require("node:child_process");
-const { join } = require("node:path");
+const { spawn } = require("child_process");
+const { join } = require("path");
```

Comment:
These exist in `examples` as well and in other places such as `node:util` in tests. I don't think they would affect users anytime soon, just a FYI